app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -299,6 +299,9 @@ export default function CompetingStep({
   ]);
 
   const guestsRestricted = competitionInfo.guest_entry_status === 'restricted';
+  const guestLimit = competitionInfo.guests_per_registration_limit != null && guestsRestricted
```

Comment:
You're in JS so this should probably be `!== null`

Reply:
Fixed!